base/settings/R/check.all.settings.R

```diff
@@ -1077,6 +1080,15 @@ check.database.settings <- function(settings) {
 #' @param settings settings file
 #' @export check.ensemble.settings
 check.ensemble.settings <- function(settings) {
+  startdate <- lubridate::parse_date_time(
+    settings$run$start.date,
+    "ymd_HMS",
+    truncated = 3)
+  enddate <- lubridate::parse_date_time(
+    settings$run$end.date,
+    "ymd_HMS",
+    truncated = 3)
+  
```

Comment:
I suggest moving this down to line 1136, next to the code that uses it.

Reply:
The current test failures seem to indicate `startdate` is winding up NA during test -- dunno if settings$run$start.date is null or in a format lubridate wasn't expecting.